### ISSUE TITLE
Plans to publish this fork on Pypi?

### ISSUE THREAD

**Issue by Lucas-C** [sarnold/pdfrw#6]
Do you plan to release this fork on Pypi?

Maybe as `pdfrw2`?

Else you may want to indicate in `README.rst` how to install your fork using `pip`:

    pip install git+https://github.com/sarnold/pdfrw.git@master

**Comment by brokenshield** [comment]
Please do! Your forked version fixed many of the bugs I was running into with 0.4 including corrupted PDF forms.

**Comment by t-houssian** [comment]
@brokenshield This forked version helped me too so I uploaded a version of it https://pypi.org/project/pdfrw2/

**Comment by Lucas-C** [comment]
@t-houssian Your `pdfrw2` Pypi homepage is a bit misleading...
It does not indicate the `git` repository of your fork, only the original one by Patrick Maupin

**Comment by t-houssian** [comment]
@Lucas-C True, Thanks! Went in and fixed it. https://pypi.org/project/pdfrw2/

**Comment by se0siris** [comment]
The PyPI page for pdfrw2 has downloads for a 0.5 version, but I'm not seeing those changes here. Was the release created from another repo?

**Comment by abubelinha** [comment]
So what's the proper way to go if I had already installed "the original" `pdfrw` like this?
`pip install pdfrw`

Can I just upgrade it this way?
`pip install pdfrw2`

Or would I run into problems because of having two different packages installed with the same "import name"?
An example of how to import and run a simple test code would be great @t-houssian @sarnold 

Thanks a lot in advance
@abubelinha

**Comment by stefan6419846** [comment]
I think it is out of scope here to explain/document basic Python packaging stuff in the package itself.

Nevertheless, installing different packages with the same namespace will work, but might leave you with an unclear result, as the installation order might affect which package code you will actually end up with. When uninstalling one of the packages, you might end up removing the actual library code the first time, while the package metadata is still present. Additionally, general advice can be hard if *pdfrw* is used as the dependency of another package in your environment, as this might lead to unsatisfied requirements.
